### ISSUE TITLE
MAINTENANCE STATUS

### ISSUE THREAD

**Issue by danieleades** [softprops/shiplift#321]
is this still maintained?

**Comment by vv9k** [comment]
I don't think so looking at the recent activity on this repo. Last commits were my merged PRs and it was quiet some time ago and it took quite a while then to get a response.

If you're interested in an actively maintained crate there is bollard or docker-api.

**Comment by that-ambuj** [comment]
hi @vv9k, I landed on there repo from [docker docs](https://docs.docker.com/engine/api/sdk/#unofficial-libraries) which mentions shiplift as one of the unofficial docker sdks for rust. Thanks a lot for recommending bollard, it saved me a ton of time!

I'd like to make few changes so that people like me don't have to hunt for what they are looking for.

- Update Bollard as a replacement to shiplift on official docker docs. (I'm currently working on creating an issue).
- Ask @softprops to give up maintainer-ship of this project or archive it with a note forwarding users to `bollard` or `docker-api`.
